lib/liquider/markup_helper.rb

```diff
@@ -1,6 +1,6 @@
 module Liquider::MarkupHelper
-  def tokenize(source)
-    Scanner.new(TextStream.new(source, mode: :liquid)).to_enum
+  def tokenize(source, mode: :text)
+    Liquider::Scanner.new(Liquider::TextStream.new(source), mode: mode).to_enum
```

Comment:
![img_0208](https://cloud.githubusercontent.com/assets/1782845/6974381/451ee7e4-d960-11e4-8aa9-8caa4837be1b.JPG)
It looks in the lexical scope the problem must be something else.